modules/ROOT/pages/graph-data-science/index.adoc

```diff
@@ -0,0 +1,62 @@
+[[graph-data-science-aura]]
+= Graph Data Science on Aura
+:description: This page describes how to use the Graph Data Science library with Aura.
+
+There are two ways to use the link:https://neo4j.com/docs/graph-data-science/current/introduction/[Graph Data Science library] with Aura:
+
+* Create an <<aura-ds, AuraDS instance>>.
+* Create an AuraDB Professional instance with the <<aura-gds-plugin>>.
+
+[[aura-ds]]
+== AuraDS
+
+label:AuraDS-Professional[] label:AuraDS-Enterprise[]
+
+AuraDS is the fully managed version of Neo4j Graph Data Science.
+
+AuraDS instances have the following features:
+
+* Upgrades and patches are automatically applied.
+* Can be seamlessly scaled up or down.
+* Can be paused to reduce costs.
+
+=== Plans
```

Comment:
We should probably add the AuraDB plans somewhere as well.

Reply:
I think we can do this at a later stage, when you get a chance to add your changes too.